core/services/ocr2/plugins/median/config/config.go

```diff
@@ -41,5 +45,16 @@ func ValidatePluginConfig(config PluginConfig) error {
 		}
 	}
 
+	// Gas price pipeline is optional
+	if !config.GasPriceSubunitsPipelineExists() {
+		return nil
+	} else if _, err := pipeline.Parse(config.GasPriceSubunitsPipeline); err != nil {
+		return errors.Wrap(err, "invalid gasPriceSubunitsSource pipeline")
+	}
+
 	return nil
 }
+
+func (config *PluginConfig) GasPriceSubunitsPipelineExists() bool {
```

Comment:
What about something like `Has*() bool`?
```suggestion
func (config *PluginConfig) HasGasPriceSubunitsPipeline() bool {
```